### DISCUSSION TITLE
cityselect/suzhou

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#17] [Announcements]
# cityselect/suzhou

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/cityselect/suzhou.html

**Comment by Yexiaomo** [comment]
lz最后去哪了，几年后也想去苏州定居。我瓜太仓还是很有希望的，嘿嘿！

**Reply by itwanger** [reply]
你猜你猜
